util/testutil/integration/run.go

```diff
@@ -35,6 +35,7 @@ func init() {
 // Backend is the minimal interface that describes a testing backend.
 type Backend interface {
```

Comment:
This could maybe use some future refactoring if we want to make integration pkg more reusable. Currently, some of these methods are very specific for a single worker implementation (or "buildkit worker" implementation). Ideally this could be used for any backend. Now that worker implementations are public they don't need to be always defined and imported with this pkg either.

Otoh, we need to be careful that the test sides do not need a lot of boilerplate for type checking.